src/components/v5/common/ActionSidebar/partials/UserSelect/UserSelect.tsx

```diff
@@ -38,9 +41,10 @@ const UserSelect: FC<UserSelectProps> = ({ name }) => {
     HTMLButtonElement,
     HTMLDivElement
   >([isUserSelectVisible]);
+  const fieldValueFormat: string = isHexString(field.value) ? field.value : '';
```

Comment:
Why do we need this? `field.value` should be user address or undefined

Reply:
we need this because in this task user have possibility to put some value in search and then this value should be display in every needed places.

Reply:
Can you check how @CzarekDryl did this in his `TokenSelect` component? Would that solve your issue?